### PR TITLE
Rashford2018Improving Summary 1

### PR DESCRIPTION
**Pull request by kraja99**

### CODE REVIEW

**Comment by zhaoyue-zephyrus** [comment]
@marcobueso any comment on this summary?

File: _summaries/radford2018improving_1.md [hunk 1]
```diff
@@ -1,19 +1,60 @@
...
+
+Beyond performance statistics, the authors also performed further experiments and ablations. They found that transfering more decoder layers between the pre-training and fine-tuning steps increases model performance. The authors measured performance ablating the auxiliary language modeling objective as well as the pre-training step all together and found that while the auxiliary objective was useful for some tasks and harmful for others, pre-training helped improve performance across the board.
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Stacking transformer decoder blocks can provide a reasonable architecture for language modeling
+* Generative pre-training can help improve performance of autoregressive language models
+* Despite SOTA results at the time, GPT was quickly eclipsed by BERT in most non-generative settings
```
Comment (marcobueso) [inline _summaries/radford2018improving_1.md:60]: Hi!
I apologize, it seems I submitted my comments on a different submission form and they did not go through.
Overall, my review was that this is a good summary in length, and it grasps the idea of the paper. On the TL;DR section, I would add, in the words of the authors, that they achieve better performance, by using "task-aware input
transformations during fine-tuning [...] while requiring minimal changes to the model architecture."

**Review by zhaoyue-zephyrus — APPROVED** [review]
